cmd/root.go

```diff
@@ -112,18 +112,18 @@ func executePhases(cmd cobra.Command, progress *phases.Progress) error {
 		progress.PrivateKey = &cmdFlagPrivateKey
 	}
 	if progress.PrivateKey == nil {
-		key, err := phases.PrivateKey()
+		_, privKey, _, err := phases.PrivateKey()
```

Comment:
the `Register an SSH key` website endpoint waits for: `auth_ssh_private_key`, `auth_ssh_public_key` and `is_register_key_into_provider_service`, so all of these options needs to be part of the progress